src/main/java/uk/gov/pay/adminusers/app/config/AdminUsersModule.java

```diff
@@ -96,7 +96,7 @@ private JpaPersistModule jpaModule(AdminUsersConfig configuration) {
     }
 
     @Provides
-    private NotificationService provideUserNotificationService() {
+    public NotificationService provideUserNotificationService() {
```

Comment:
It seems the private keyword on the method didn't stop the app from running...

Reply:
It stops PMD complaining about an apparently unused private method.